ui/perfherder/compare/CompareTable.jsx

```diff
@@ -24,76 +57,82 @@ export default class CompareTable extends React.PureComponent {
       onPermalinkClick,
       history,
     } = this.props;
+    const { showRetriggerModal } = this.state;
 
     return (
-      <Table
-        id={getHashBasedId(testName, hashFunction)}
-        aria-label="Comparison table"
-        sz="small"
-        className="compare-table mb-0 px-0"
-        key={testName}
-        innerRef={el => {
-          this.header = el;
-        }}
-      >
-        <thead>
-          <tr className="subtest-header bg-lightgray">
-            <th className="text-left">
-              <span>{testName}</span>
-              {onPermalinkClick && (
-                <Button
-                  className="permalink p-0 ml-1"
-                  color="link"
-                  onClick={() =>
-                    onPermalinkClick(
-                      getHashBasedId(testName, hashFunction),
-                      history,
-                      this.header,
-                    )
-                  }
-                  title="Permalink to this test table"
-                  aria-label={`Permalink to test ${testName} table`}
-                >
-                  <FontAwesomeIcon icon={faHashtag} />
-                </Button>
-              )}
-            </th>
-            <th className="table-width-lg">Base</th>
-            {/* empty for less than/greater than data */}
-            <th className="table-width-sm" />
-            <th className="table-width-lg">New</th>
-            <th className="table-width-lg">Delta</th>
-            <th className="table-width-lg">Magnitude of Difference</th>
-            <th className="table-width-lg">Confidence</th>
-            <th className="text-right table-width-md">
-              {hasSubtests &&
-                data &&
-                data.length &&
-                (data[0].newRetriggerableJobId || !isBaseAggregate) &&
-                user.isLoggedIn && (
+      <React.Fragment>
+        <RetriggerModal
+          showModal={showRetriggerModal}
+          toggle={() => this.toggle('showRetriggerModal')}
```

Comment:
Do not use arrow functions when rendering. 
`Note:
Using an arrow function in render creates a new function each time the component renders, which may break optimizations based on strict identity comparison.
`([Reference](https://reactjs.org/docs/faq-functions.html))